doc/man/date-of-man-include.am

```diff
@@ -1,10 +1,10 @@
 SED_PARAMETERS = \
 	-e "s/%DATE_OF_MAN_PAGE%/$${MAN_DATE}/g" \
-	-e "s/%MAN_VERSION%/@MAN_VERSION@/g" \
-	-e "s{%sysconfdir%{@sysconfdir@{g" \
-	-e "s{%libexecdir%{@libexecdir@{g" \
-	-e "s{%pkglibexecdir%{@pkglibexecdir@{g" \
-	-e "s{%pkgdatadir%{$(datadir)/@PACKAGE@{g"
+	-e "s/%MAN_VERSION%/$(MAN_VERSION)/g" \
+	-e "s{%sysconfdir%{$(sysconfdir){g" \
+	-e "s{%libexecdir%{$(libexecdir){g" \
+	-e "s{%pkglibexecdir%{$(pkglibexecdir){g" \
+	-e "s{%pkgdatadir%{$(pkgdatadir){g"
```

Comment:
LGTM, that's better.
I think the reason why it didn't work for me was that I used curly braces (`${FOO}`) which is only seen by the shell..
And also curly braces probably brace the sed command because weirdly, `{` is used as separator.
I was confused by the `$${MAN_DATE}` above.. I don't understand why we're delaying expansion of `MAN_DATE`, maybe because it makes a small difference